Resources/Prototypes/_NF/Shipyard/monolith.yml

```diff
@@ -0,0 +1,32 @@
+- type: vessel
+  id: MONOLITH
+  name: KL Monolith
+  description: A large and slow universal ship designed for manual upgrades by the crew.
+  price: 80000 #68k before +15% 
+  category: Large
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/monolith.yml
+
+- type: gameMap
+  id: MONOLITH
+  mapName: 'KL Monolith'
+  mapPath: /Maps/Shuttles/monolith.yml
+  minPlayers: 0
+  stations:
+    MONOLITH:
```

Comment:
same here, do not do this. You will also need to change this if you did it on the BecomesStation component.